spec/acceptance/nodesets/default.yml

```diff
@@ -1,17 +1,55 @@
 ---
```

Comment:
This file is added for testing purposes only. When it comes time to merge the PR I can remove this and revert it back to default.